WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorViewModel.swift

```diff
@@ -42,17 +42,20 @@ final class CustomFieldEditorViewModel: ObservableObject {
         initialKey == "" && initialValue == ""
     }
 
-    init(key: String,
-         value: String,
-         isJsonField: Bool = false,
+    init(customField: CustomFieldViewModel?,
          disallowedKeys: [String] = [],
          onSave: @escaping (String, String) -> Void,
          onDelete: (() -> Void)? = nil) {
-        self.key = key
-        self.value = isJsonField ? value.prettyPrint() : value
-        self.initialKey = key
-        self.initialValue = isJsonField ? value.prettyPrint() : value
-        self.isReadOnlyValue = isJsonField
+        self.key = customField?.key ?? ""
+        self.initialKey = customField?.key ?? ""
+        let value = if customField?.isJson ?? false, let value = customField?.value {
```

Comment:
Maybe this shows the intent better? 

```suggestion
        let value = if customField?.isJson == true, let value = customField?.value {
```

Reply:
Thanks, applied.
For context, I looked online, and there doesn't seem to be a consensus on the best syntax here unlike Kotlin, so I'm applying your suggestion if you think it's easier to read.